src/routes/registration.go

```diff
@@ -12,12 +11,11 @@ import (
 
 func (a *APIService) addRegistrationRoutes(e *echo.Echo) {
 	api := e.Group("/v1")
-	api.POST("/peers", a.peerUpdate)
 	api.POST("/join_network", a.joinNetwork)
 }
 
 func (a *APIService) joinNetwork(c echo.Context) error {
-	log.Println("[master] Microcontroller asked to join cluster")
+	log.Println("Device asked to join cluster")
```

Comment:
This route only seems necessary if the devices know where DeviceCommander is running. I suppose the devices could always attempt to connect to the last known DeviceCommander and this would provide a more fluid experience of known devices auto connecting. And actually now that i think about it is prolly desired in case of accidental device restarts. So yeah carry on sir!